docs/operator/migration.md

```diff
@@ -48,12 +48,12 @@ See [Custom Resource Scenarios](../../deploy/scenarios) for examples or refer to
-| `proxy.histogramDayPort`           | `dataExport.wavefrontProxy.histogram.dayPort`                                                      | Port to accumulate 1-day based histograms on Wavefront data format (usually 40002)                                                                             |
-| `proxy.deltaCounterPort`           | `dataExport.wavefrontProxy.deltaCounterPort`                                                       | Port to send delta counters on Wavefront data format (usually 50000)                                                                                           |
-| `proxy.args`                       | `dataExport.wavefrontProxy.args`                                                                   | Additional Wavefront proxy properties can be passed as command line arguments in the `--<property_name> <value>` format. Multiple properties can be specified. |
+| `proxy.histogramPort`              | `dataExport.wavefrontProxy.histogram.port`                                                         | Port for Operations for Applications histogram distributions (usually 40000)                                                                                   |
+| `proxy.histogramMinutePort`        | `dataExport.wavefrontProxy.histogram.minutePort`                                                   | Port to accumulate 1-minute based histograms on Operations for Applications data format (usually 40001)                                                        |
+| `proxy.histogramHourPort`          | `dataExport.wavefrontProxy.histogram.hourPort`                                                     | Port to accumulate 1-hour based histograms on Operations for Applications data format (usually 40002)                                                          |
+| `proxy.histogramDayPort`           | `dataExport.wavefrontProxy.histogram.dayPort`                                                      | Port to accumulate 1-day based histograms on Operations for Applications data format (usually 40002)                                                           |
+| `proxy.deltaCounterPort`           | `dataExport.wavefrontProxy.deltaCounterPort`                                                       | Port to send delta counters on Operations for Applications data format (usually 50000)                                                                         |
+| `proxy.args`                       | `dataExport.wavefrontProxy.args`                                                                   | Additional Wavefront Proxy properties can be passed as command line arguments in the `--<property_name> <value>` format. Multiple properties can be specified. |
```

Comment:
Additional Wavefront proxy properties can be passed as command-line arguments in the `--<property_name> <value>` format. Multiple properties can be specified.